packages/gio-identify/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+let gio = require('../gio/index');
+let load = require('../gio-load/index');
+let convert_geometry = require('../gio-convert-geometry/index');
+
+module.exports = (image, geometry) => {
+		
+	// convert point
+	let point = convert_geometry('point', geometry);
+	let lng = point[0];
+	let lat = point[1];
+
+	// convert image
+	let fd = image.fileDirectory;
+	let geoKeys = image.getGeoKeys();
+
+	// TEMPORARY: make sure raster is in wgs 84
+	if (geoKeys.GeographicTypeGeoKey === 4326 && geoKeys.GeogCitationGeoKey === "WGS 84") {
+		
+		let data = image.readRasters();
```

Comment:
You'll want to use rasterWindow here.  Will still probably accept PR request, because certainly a million times better than nothing!